Makefile

```diff
@@ -1,67 +1,75 @@
-# A Self-Documenting Makefile: http://marmelab.com/blog/2016/02/29/auto-documented-makefile.html
+# The version of this client (should be in line with the highest supported engine/enterprise version
```

Comment:
does this imply that the engine/enterprise version are correlated? not sure I follow this comment

Reply:
good question. I put in the option for a version here that we can set, but don't have a clear idea of how it should be used yet (since it's really the swagger version that dictates this info).